pkg/trantor/testing/smr_test.go

```diff
@@ -188,8 +187,8 @@ func testIntegrationWithISS(tt *testing.T) {
 		13: {"Submit 100 fake transactions with 4 nodes in simulation, two of them crashed (not messages sent, yes received) and holding the supermajority of stake",
 			&TestConfig{
 				NodeIDsWeight: deploytest.NewNodeIDsWeights(4, func(id t.NodeID) types.VoteWeight {
-					idfloat, _ := strconv.ParseFloat(id.Pb(), 64)
-					return types.VoteWeight(math.Pow(2, idfloat)) // ensures last 2 nodes weight is greater than twice the sum of the others'
+					numericID, _ := strconv.ParseInt(id.Pb(), 10, 64)
```

Comment:
not sure this is strictly needed (along pow2 function). AFAIS this is still uint64 (and fits in uint64), and using float would allow us to use the native math.pow. If we had this in BigInts then it would make more sense for me. But minor thing, if you have a good reason for this change, let us leave it.

Reply:
The main reason I wrote the custom `pow2` function for ints is that i've always been weary of converting floats to ints, even if they "should" be integers in the mathematical sense. Otherwise I'd have stuck with floats and just appended the string zeroes as I do now anyway.